redux/longestStreakSlice.ts

```diff
@@ -36,7 +35,9 @@ const initialState: LongestStreakState = {
   stage: STAGE.SET_RULES,
   reset: false,
   level: GameLevel.BEGINNER,
-  blocks: initializeGameState(GameLevel.INTERMEDIATE),
+
+  blocks: initializeGameState(),
```

Comment:
so the game will always initialize as a beginner, but later we will fetch the users level from the database and update the game state as needed?

Reply:
yes. using graph ql queries and mutations